src/Paramore.Brighter.MessagingGateway.AzureServiceBus/AzureServiceBusConsumer.cs

```diff
@@ -19,9 +19,12 @@ public class AzureServiceBusConsumer : IAmAMessageConsumer
         private bool _subscriptionCreated;
         private static readonly Lazy<ILog> _logger = new Lazy<ILog>(LogProvider.For<AzureServiceBusConsumer>);
         private readonly OnMissingChannel _makeChannel;
+        private readonly ReceiveMode _receiveMode;
+
+        private const string _lockTokenKey = "LockToken";
         
         public AzureServiceBusConsumer(string topicName, string subscriptionName, IAmAMessageProducer messageProducer, IManagementClientWrapper managementClientWrapper, 
-            IMessageReceiverProvider messageReceiverProvider, int batchSize = 10, OnMissingChannel makeChannels = OnMissingChannel.Create)
+            IMessageReceiverProvider messageReceiverProvider, int batchSize = 10, OnMissingChannel makeChannels = OnMissingChannel.Create, ReceiveMode receiveMode = ReceiveMode.ReceiveAndDelete)
```

Comment:
Do you want to move this parameter above OnMissingChannel in the list. I think that should always be the last parameter.

Reply:
@iancooper Have made this change and updated the PR